<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #00FFFF;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>532</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="1">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_7">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"> background: transparent; /* устанавливаем прозрачный фон */
    border: 1px solid black; /* устанавливаем черные границы */
    color: black; 
  border-radius: 10%;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButton_8">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"> background: transparent; /* устанавливаем прозрачный фон */
    border: 1px solid black; /* устанавливаем черные границы */
    color: black; 
  border-radius: 10%;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_6">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"> background: transparent; /* устанавливаем прозрачный фон */
    border: 1px solid black; /* устанавливаем черные границы */
    color: black; 
  border-radius: 10%;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_3">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"> background: transparent; /* устанавливаем прозрачный фон */
    border: 1px solid black; /* устанавливаем черные границы */
    color: black; 
  border-radius: 10%;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="autoRepeatDelay">
          <number>300</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_5">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"> background: transparent; /* устанавливаем прозрачный фон */
    border: 1px solid black; /* устанавливаем черные границы */
    color: black; 
  border-radius: 10%;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="pushButton_9">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"> background: transparent; /* устанавливаем прозрачный фон */
    border: 1px solid black; /* устанавливаем черные границы */
    color: black; 
  border-radius: 10%;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_2">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"> background: transparent; /* устанавливаем прозрачный фон */
    border: 1px solid black; /* устанавливаем черные границы */
    color: black; 
  border-radius: 10%;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background: transparent; /* устанавливаем прозрачный фон */
    border: 1px solid black; /* устанавливаем черные границы */
    color: black; 
  border-radius: 10%;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_4">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"> background: transparent; /* устанавливаем прозрачный фон */
    border: 1px solid black; /* устанавливаем черные границы */
    color: black; 
  border-radius: 10%;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>10</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"> background: transparent; /* устанавливаем прозрачный фон */
    border: 1px solid black; /* устанавливаем черные границы */
    color: black; 
  border-radius: 10%;
</string>
       </property>
       <property name="title">
        <string>Choose X or O</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>181</width>
          <height>67</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <italic>true</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">border_radius:5%;
    border: 0px solid black;</string>
             </property>
             <property name="text">
              <string>Player 1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="player1X">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">border-radius:5%;
    border: 0px solid black;</string>
             </property>
             <property name="text">
              <string>X</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="Player1O">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <italic>false</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">    border: 0px solid black;</string>
             </property>
             <property name="text">
              <string>O</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <italic>true</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">    border: 0px solid black;</string>
             </property>
             <property name="text">
              <string>Player 1 wins</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="labelPlayer1Wins">
             <property name="minimumSize">
              <size>
               <width>41</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>41</width>
               <height>27</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font-size: 14px;
font-weight: bold;
font-style: italic;
text-align: center;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>190</y>
          <width>181</width>
          <height>67</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <italic>true</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">    border: 0px solid black;</string>
             </property>
             <property name="text">
              <string>Player 2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="player2X">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">    border: 0px solid black;</string>
             </property>
             <property name="text">
              <string>X</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="player2O">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">    border: 0px solid black;</string>
             </property>
             <property name="text">
              <string>O</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <italic>true</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">    border: 0px solid black;</string>
             </property>
             <property name="text">
              <string>Player 2 wins</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="labelPlayer2Wins">
             <property name="minimumSize">
              <size>
               <width>41</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>41</width>
               <height>27</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font-size: 14px;
font-weight: bold;
font-style: italic;
text-align: center;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
